src/pages/Claim/StartNewClaimSection.tsx

```diff
@@ -6,22 +6,19 @@ import { Text } from "../../components/Text"
 import { Protocol } from "../../hooks/api/protocols"
 import { Column } from "../../components/Layout"
 import { NewClaimModal } from "./NewClaimModal"
-import { ethers } from "ethers"
 
 type Props = {
   protocol: Protocol
 }
 
 export const StartNewClaimSection: React.FC<Props> = ({ protocol }) => {
-  const [{ data: connectedAccount }] = useAccount()
+  const { address: connectedAddress } = useAccount()
   const [isCreating, setIsCreating] = useState(false)
   const [canStartNewClaim, setCanStartNewClaim] = useState(false)
 
   useEffect(() => {
-    setCanStartNewClaim(
-      !!connectedAccount?.address && ethers.utils.getAddress(connectedAccount.address) === protocol.agent
-    )
-  }, [connectedAccount?.address, protocol.agent])
+    setCanStartNewClaim(!!connectedAddress && connectedAddress === protocol.agent)
```

Comment:
Here you are using double negation to check if `connectedAddress` has a *truthy* value, but in the other files, you did not check this. It's redundant, the comparison will still work without this check, but wanted to know if there should be a difference between this file and the others.

Reply:
Nope, you're right. I can remove the first check.